.travis.yml

```diff
@@ -8,17 +8,15 @@ virtualenv:
 # This (sudo: false) is needed to "run on container-based infrastructure" on
 # which cache: is available
 # http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+dist: trusty
 sudo: false
 
 cache: pip
 
 addons:
   apt_packages:
-    # Although we are installing a custom Ice 3.5 build, this package is still
-    # required to install some of the runtime Ice dependencies
-    - ice34-services
-    - python-zeroc-ice
-  postgresql: "9.3"
+    - zeroc-ice35
```

Comment:
This will not work due to a problem on travis at the moment see https://github.com/openmicroscopy/openmicroscopy/pull/5409
for the change required